### PR TITLE
[Storage] Support Failover, and blob Service

### PR DESCRIPTION
**Pull request by blueww**
## Description

Swagger PR:

- BlobService (already merged): https://github.com/Azure/azure-rest-api-specs/pull/4316
- Failover PR (in review): https://github.com/Azure/azure-rest-api-specs/pull/4457
- Get LastSyncTime PR (in review): https://github.com/Azure/azure-rest-api-specs/pull/4455

The SDK is currently generated from a temp repo with contains all the above swagger PR change.
Will regenerate it against public swagger when all the swagger PR merged.

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] Please add REST spec PR link to the SDK PR
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [x] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [x] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Review by dsgouda — CHANGES_REQUESTED** [review]
Looks good for the most part, please regenerate the code when spec is merged

File: src/SDKs/Storage/Management.Storage/Generated/SdkInfo_StorageManagementClient.cs
Comment (blueww) [reply src/SDKs/Storage/Management.Storage/Generated/SdkInfo_StorageManagementClient.cs]: @dsgouda 
I have updated the PR per the public swagger, and run msbuild build.proj (Anyway, don't see any change by run msbuild).
Would you please help to review and merge the PR asap?
Thanks!

Comment (dsgouda) [reply src/SDKs/Storage/Management.Storage/Generated/SdkInfo_StorageManagementClient.cs]: @blueww Can't merge the PR until spec PR is merged.
This PR looks okay, please regenerate the code when all spec PRs are merged

File: src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj
Comment (dsgouda) [inline src/SDKs/Storage/Management.Storage/Microsoft.Azure.Management.Storage.csproj]: @blueww When are you planning to release a stable version? Ideally you should have started preview versioning from `8.8.0-preview` and landed on `9.0.0` stable version. Do let us know when you are planning to move to version 10 so we can provide a better idea.
Not blocking on this

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**
